stopcovid/event_distributor/event_distributor.py

```diff
@@ -0,0 +1,18 @@
+from stopcovid.utils import dynamodb as dynamodb_utils
+
+
+from stopcovid.dialog.dialog import event_from_dict
+
+from stopcovid.event_distributor.outbound_sms import distribute_outbound_sms_events
+
+
+def distribute_dialog_events(event, context):
+    dialog_events = [
+        event_from_dict(dynamodb_utils.deserialize(record["dynamodb"]["NewImage"]))
+        for record in event["Records"]
+        if record["dynamodb"].get("NewImage")
+    ]
+
+    distribute_outbound_sms_events(dialog_events)
```

Comment:
so, you are converting and localizing in the distributor. Then the SMS sender just sends? I think this is right.

A crash in the localization logic would block the entire stream for all consumers. But TBH it probably should.

Reply:
Yeah, my thinking was that the SMS sender should be as dumb as possible, while business logic lives in the event distributor.